### PR TITLE
feature(portfolio): update token activity api response type

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Title says

##### Ticket

YOMO-1855

### CODE REVIEW

File: packages/portfolio/src/adapters/dullahan-api/token-activity.mocks.ts [hunk 1]
```diff
@@ -47,10 +47,10 @@ const ftNameless = {
 
 const apiResponseSuccessDataOnly: Readonly<DullahanApiTokenActivityUpdatesResponse> =
   {
-    [tokenInfoMocks.primaryETH.id]: primaryETH,
-    [tokenInfoMocks.rnftWhatever.id]: rnftWhatever,
-    [tokenInfoMocks.ftNoTicker.id]: ftNoTicker,
-    [tokenInfoMocks.ftNameless.id]: ftNameless,
+    [tokenInfoMocks.primaryETH.id]: [200, primaryETH],
```
Comment (stackchain) [inline packages/portfolio/src/adapters/dullahan-api/token-activity.mocks.ts]: I think there is a `Api.HttpsStatus.Ok`

File: packages/portfolio/src/adapters/dullahan-api/types.ts [hunk 2]
```diff
@@ -35,6 +35,11 @@ export type DullahanApiTokenActivityUpdates = Readonly<{
   price24h: DullahanApiTokenActivityRecord
 }>
 
+export type DullahanApiTokenActivityUpdatesStatusCode = number
```
Comment (stackchain) [inline packages/portfolio/src/adapters/dullahan-api/types.ts]: build the records `[HttpStatus.OK, Portfolio.TokenActivity] | [404 | 500, string]`

File: packages/portfolio/src/adapters/dullahan-api/api-maker.ts [hunk 3]
```diff
@@ -305,7 +306,8 @@ export const apiConfig: ApiConfig = freeze(
       tokenInfo: 'https://zero.yoroiwallet.com/tokens/info',
       tokenInfos: 'https://zero.yoroiwallet.com/tokens/info/multi',
       tokenTraits: 'https://zero.yoroiwallet.com/tokens/nft/traits',
-      tokenActivityUpdates: 'https://zero.yoroiwallet.com/defi/token-activity',
```
Comment (banklesss) [inline packages/portfolio/src/adapters/dullahan-api/api-maker.ts:308]: created this ticket to update the endpoints later: https://emurgo.atlassian.net/browse/YOMO-1858

File: packages/portfolio/src/adapters/dullahan-api/transformers.test.ts [hunk 4]
```diff
@@ -107,12 +107,14 @@ describe('transformers', () => {
   })
 
   describe('toTokenActivityUpdates', () => {
-    it('should return an empty object if apiTokenActivityUpdates is empty', () => {
+    it('should return an empty object if apiTokenActivityUpdates ststus code is not empty', () => {
```
Comment (banklesss) [inline packages/portfolio/src/adapters/dullahan-api/transformers.test.ts]:
```suggestion
    it('should throw an error if apiTokenActivityUpdates response is empty', () => {
```

File: packages/portfolio/src/adapters/dullahan-api/api-maker.ts (hunk 3 above)
Comment (banklesss) [reply packages/portfolio/src/adapters/dullahan-api/api-maker.ts:308]: done

**Review by michalszorad — APPROVED** [review]

**Pull request merged**